services/workflows-service/src/business-report/business-report.service.ts

```diff
@@ -160,26 +159,18 @@ export class BusinessReportService {
 
         const businessWithRequests = await Promise.all(businessCreatePromises);
 
-        const businessReportRequests = businessWithRequests.map(
-          ({ businessReportRequest, businessId }) => {
-            return {
-              withQualityControl,
-              websiteUrl: businessReportRequest.websiteUrl,
-              lineOfBusiness: businessReportRequest.lineOfBusiness,
-              parentCompanyName: businessReportRequest.parentCompanyName,
-              callbackUrl: `${env.APP_API_URL}/api/v1/internal/business-reports/hook?businessId=${businessId}`,
-              merchantId: businessId,
-              customerId,
-            };
-          },
-        ) satisfies TReportRequest;
-
         await this.merchantMonitoringClient.createBatch({
-          reportRequests: businessReportRequests,
-          clientName: 'merchant',
-          reportType: type,
-          withQualityControl,
+          customerId,
           workflowVersion,
+          withQualityControl,
+          reportType: type,
+          reports: businessWithRequests.map(({ businessReportRequest, businessId }) => ({
+            businessId,
+            websiteUrl: businessReportRequest.websiteUrl,
+            countryCode: businessReportRequest.countryCode,
+            parentCompanyName: businessReportRequest.parentCompanyName,
+            callbackUrl: `${env.APP_API_URL}/api/v1/internal/business-reports/hook?businessId=${businessId}`,
```

Comment:
We need the callback URL?

Reply:
the callback url should be related per each busniess 
so we can set this property optional and remove it from the plugin

Reply:
No, currently we don't need callback URL, the endpoint doesn't do anything.